src/Dodo.HttpClient.ResiliencePolicies/HttpClientSettings.cs

```diff
@@ -8,16 +8,18 @@ public class HttpClientSettings
 	{
 		public TimeSpan HttpClientTimeout { get; }
 		public TimeSpan TimeoutPerTry { get; }
+		public TimeSpan? TimeoutOverall { get; }
```

Comment:
Do we really have to make it nullable? It adds unnecessary complexity to the code and to the usage. Actually the reasonable defaults looks better I think, because the main goal of the library is to simplicity.

Reply:
Maybe it is good idea to leave **only** TimeoutOverall and TimeoutPerTry? For the HttpClient we can calculate Timeout like this: `Timeout = TimeoutOverall + delta`, where `delta` is some hard-coded constant time. We need it just because some policies like Fallback or Caching may sit between TimeoutPolicy and Client itself.
Yes it is a breaking change, but we can do it because it is a library and we can change major version for such changes. It also allows us to get rid of nulls and make our code much cleaner.

Reply:
I was thinking about preservation backward compatibility.
So I decided to both introduce a new parameter and make it optional.

The idea of unifying the parameter looks good.

Reply:
You may not think about backward compatibility. Just change PR base from `master` to `milestone/v2.0.0`. We will include all breaking changes to new major version of the library.